tests/translate/translate_neg_adj3.py

```diff
@@ -33,3 +33,23 @@ def __init__(self, grid):
         }
         for qvar in utils.tracer_variables:
             self.ignore_near_zero_errors[qvar] = True
+
+    def compute(self, inputs):
+        self.make_storage_data_input_vars(inputs)
+        compute_fn = AdjustNegativeTracerMixingRatio(
```

Comment:
Not a big deal, but curious why not just set `self.compute_func` to `AdjustNegativeTracerMixingRatio(...)`?

Reply:
His init routine relied on model data, qvapor and qgraupel, which isn't known at init time of the translate class. Otherwise I'd suggest the same thing!